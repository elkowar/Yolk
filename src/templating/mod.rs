pub mod document;
pub mod element;
mod parser;

pub(crate) const COMMENT_START: &str = "<yolk> ";
