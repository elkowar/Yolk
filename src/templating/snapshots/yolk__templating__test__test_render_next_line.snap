---
source: src/templating/mod.rs
expression: "/* {# string.upper(YOLK_TEXT) #} */\nfoo\n"
snapshot_kind: text
---
/* {# string.upper(YOLK_TEXT) #} */
FOO
